src/main/java/testing_of_mobile/service/PropertyForMobileFactory.java

```diff
@@ -11,12 +11,21 @@
 
     private PropertyForMobileFactory() {}
 
-    public static DesiredCapabilities getCapabilities() {
+    public static DesiredCapabilities getCapabilitiesOnliner() {
         DesiredCapabilities capabilities = new DesiredCapabilities();
         capabilities.setCapability("deviceName", PropertyReader.getData(DEVICE_NAME));
         capabilities.setCapability("platformName", PropertyReader.getData(PLATFORM_NAME));
         File app = new File("src/main/resources/onliner.apk");
         capabilities.setCapability("app", app.getAbsolutePath());
         return capabilities;
     }
+
+    public static DesiredCapabilities getCapabilitiesBracket() {
```

Comment:
Looks like code duplication, please extract common code to a separate method with a parameter